resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -723,6 +723,25 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Maximum number of pending pods should be a positive integer")
       .createWithDefault(Int.MaxValue)
 
+  val KUBERNETES_MAX_EXECUTOR_FAILURES =
+    ConfigBuilder("spark.kubernetes.executor.maxNumFailures")
```

Comment:
The equivalent yarn configuration is `spark.yarn.max.executor.failures`

Reply:
How about `spark. kubernetes.max.executor.failures`? Keep same name  pattern as `yarn`